schema/release-schema.json

```
@@ -27,13 +27,13 @@
     },
     "publisher": {
       "title": "Publisher",
-      "description": "The original publisher of this release.",
+      "description": "The original publisher of the release. For a compiled release this may differ from the publisher(s) of the individual releases about the contracting process.",
```

Use non-normative synonym. Add "or planning process".

```suggestion
      "description": "The original publisher of the release. The publisher of a compiled release for a contracting or planning process might differ from the publishers of its releases.",
```

---

But the schema is supposed to be normative?

---

I think the first sentence sets the normative semantics, and the second sentence provides some additional guidance to aid interpretation. I might be wrong, though!